easy_music_generator/tests/test_easy_music_generator.py

```diff
@@ -0,0 +1,29 @@
+import os
+import glob
+import unittest
+import easy_music_generator as emg
+
+
+class TestEasyMusicGenerator(unittest.TestCase):
+    """
+    Unit tests for easy_music_generator.py
+    """
+
+    def setUp(self):
+
+        pre_exist = glob.glob('./output/*.mid')
+
+        if len(pre_exist):
+            for midi in pre_exist:
+                os.remove(midi)
+
+        if os.path.exists("./output"):
+            os.rmdir("./output")
+
+    def test_easy_music_generator(self):
+        emg_obj = emg.EasyMusicGenerator()
```

Comment:
Add docstring explaining what this function is doing. 
'''
Tests that EasyMusicGenerator is outputting midi files by checking that the output folder has more than zero files midi.
'''